tests/test_actions.py

```diff
@@ -895,21 +853,25 @@ def test_passthru_attribute_missing(self) -> None:
 
     def test_answerable_answers(self, Tester) -> None:
         question = FakeQuestion()
-        Silently(question).answered_by(Tester)
+        original_answered_by = question.answered_by
+        s = Silently(question)
+        s.answered_by(Tester)
```

Comment:
This line should be on its own as the "Action" section of the test, like this:
```python
    def test_answerable_answers(self, Tester) -> None:
        question = FakeQuestion()
        original_answered_by = question.answered_by
        s = Silently(question)

        s.answered_by(Tester)

        original_answered_by.assert_called_once_with(Tester)
```

I wouldn't have said anything, but i already asked you to edit this file anyway. :P

Reply:
Also it looks like this test changed the format for some reason, but the tests below stayed the same. Can you make that a little more consistent?

What i mean is, this one now does `s = Silently(question)` while the others kept the `Silently(action).perform_as(Tester)` arrangement.

Reply:
fixed